lib/sles4sap.pm

```diff
@@ -951,7 +951,8 @@ sub do_hana_takeover {
     }
     sleep bmwqemu::scale_timeout(10);
     if ($args{cluster}) {
-        assert_script_run "crm resource cleanup rsc_SAPHana_${sid}_HDB$instance_id", $args{timeout};
+        my $hana_resource = get_var('USE_SAP_HANA_SR_ANGI') ? "rsc_SAPHanaCtl_${sid}_HDB${instance_id}" : "rsc_SAPHana_${sid}_HDB${instance_id}";
```

Comment:
Just a note for myself:

- `rsc_SAPHanaCtl_${sid}_HDB${instance_id}` is the name of a resource type  `ocf:suse:SAPHanaController` from the new file [data/sles4sap/angi_hana_cluster.conf](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19867/files#diff-7a35318b3b4c3de3f8a63ea45ac54d6cba9b83b68913c960152f2309814026d8)

- `rsc_SAPHana_${sid}_HDB${instance_id}` is from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/dfb5105b8c95acd998ae48278bd9cc3e449159c1/data/sles4sap/hana_cluster.conf#L6 and refer to a resource of type `ocf:suse:SAPHana`